zimbra/src/main/java/fr/openent/zimbra/model/task/ICalTask.java

```diff
@@ -35,7 +35,16 @@ public ICalTask(JsonObject icalRequest) throws Exception {
 
     public ICalTask(Action<ICalTask> action, JsonObject icalRequest) throws Exception {
         super(icalRequest, action);
-        new ICalTask(icalRequest);
+
+        try {
+            this.id = icalRequest.getLong(Field.ID, null);
+            this.actionId = icalRequest.getLong(Field.ACTION_ID, null);
+            this.status = TaskStatus.fromString(icalRequest.getString(Field.STATUS, null));
+            this.name = icalRequest.getString(Field.NAME, null);
+            this.body = new JsonObject(icalRequest.getString(Field.BODY, "{}"));
```

Comment:
Possibilité d'utiliser une constante ici pour represénter le string de Json vide (meilleure lisibilité du code)

Reply:
this.body = new JsonObject(icalRequest.getString(Field.BODY), new JsonObject());

Comme ça on met tout le monde d'accord ^^